src/SprykerEco/Zed/Unzer/Business/Credentials/UnzerCredentialsCreator.php

```diff
@@ -114,6 +125,10 @@ protected function executeCreateUnzerCredentials(UnzerCredentialsTransfer $unzer
             $unzerCredentialsTransfer->getUnzerKeypairOrFail()->getPrivateKeyOrFail(),
         );
 
+        if (in_array($unzerCredentialsTransfer->getTypeOrFail(), UnzerConstants::UNZER_CHILD_CONFIG_TYPES, true)) {
```

Comment:
required prtoperties should be mentioned at API specs

Reply:
fixed